### DISCUSSION TITLE
Fuzzy stars in Stellarium 0.21.3?

### DISCUSSION THREAD

**Discussion by pchapin** [Stellarium/stellarium#2198] [Q&A]
I just installed Stellarium 0.21.3 on my Windows 11 laptop. I'm a new user. Overall, the program appears to run fine, but I noticed when I zoom in, the stars in certain regions of the display are fuzzy, as they might appear in an out-of-focus telescope (fuzzy disks around a brighter central dot). The curious thing is that the effect is only apparent for parts of the display, in other parts the stars are perfectly clear. Is this a known effect? It would be far better if it didn't do this.

I did download star catalog #4, if that matters.

**Comment by gzotti** [comment]
All stars are displayed as round disk with fuzzy edge. Alternatively, spiky stars can be selected in the view settings.

**Reply by pchapin** [reply]
What I'm seeing is different because the effect varies from place to place on the screen. I think Stellarium is trying to show the stars embedded in a sprawling reflection nebula. I notice the effect in places like Orion where such nebulae are extensive. Is there a way of turning the rendering of nebula off? It's distracting and a bit ugly when I'm trying to use Stellarium as a star atlas.

Thanks!

**Reply by alex-w** [reply]
Hint: press buttons I and M

The list of useful shortcuts available in Help dialog (F1) and in Stellarium User Guide

**Reply by pchapin** [reply]
@alex-w Thanks for that hint, I found what I needed! I had looked over those shortcuts before, but didn't notice the right one to use... there are a lot!
